docs/doc/14-sql-commands/00-ddl/91-catalog/create-catalog.md

```diff
@@ -0,0 +1,58 @@
+---
+title: CREATE CATALOG
+---
+
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.65"/>
+
+Defines and establishes a new catalog in the Databend query engine. 
+
+## Syntax
+
+```sql
+CREATE CATALOG <catalog_name>
+TYPE = <catalog_type>
+CONNECTION = (
+    ADDRESS = '<hive_metastore_address>'
+    URL = '<data_storage_path>'
+    <connection_parameter> = '<connection_parameter_value>'
+    <connection_parameter> = '<connection_parameter_value>'
+    ...
+)
+```
+
+| Parameter             | Description                                                                                                               |
+|-----------------------|---------------------------------------------------------------------------------------------------------------------------|
+| TYPE                  | Type of the catalog: 'HIVE' for Hive catalog or 'ICEBERG' for Iceberg catalog.                                      |
+| ADDRESS               | Address of the Hive Metastore. Required for Hive catalog.                                                                |
+| URL                   | Location of the external storage linked to this catalog. This could be a bucket or a folder within a bucket. For example, 's3://databend-toronto/'.                       |
+| connection_parameter  | Connection parameters to establish connections with external storage. The required parameters vary based on the specific storage service and authentication methods. Refer to [Connection Parameters](../../../13-sql-reference/51-connect-parameters.md) for detailed information. |
+
+## Examples
+
+This example demonstrates the creation of a catalog configured to interact with the Hive Metastore and access data stored on Amazon S3, located at 's3://databend-toronto/'.
+
+```sql
+CREATE CATALOG hive_ctl 
+TYPE = HIVE 
+CONNECTION =(
+    ADDRESS = '127.0.0.1:9083' 
+    URL = 's3://databend-toronto/' 
+    AWS_KEY_ID = '<your_key_id>' 
+    AWS_SECRET_KEY = '<your_secret_key>' 
+);
+```
+
+This example demonstrates the creation of a catalog configured to interact with an Iceberg data storage located in MinIO at 's3://databend/iceberg/'.
+
+```sql
+CREATE CATALOG iceberg_ctl
+TYPE = ICEBERG
+CONNECTION = (
+    URL = 's3://databend/iceberg/'
+    AWS_KEY_ID = 'minioadmin'
+    AWS_SECRET_KEY = 'minioadmin'
+    ENDPOINT_URL = 'http://127.0.0.1:9000'
```

Comment:
The `ENDPOINT_URL` is not show in the syntax.
Here is more confuse with: `ADDRESS` and `ENDPOINT_URL`, it's hard for the user to choose when to use.
@Xuanwo @PsiACE

Reply:
ENDPOINT_URL is a connection_parameter in the syntax: https://databend.rs/doc/sql-reference/connect-parameters

Reply:
> Here is more confuse with: `ADDRESS` and `ENDPOINT_URL`, it's hard for the user to choose when to use, why we combine them into one, like `ADDRESS`?

`ADDRESS` refers to the address for the hive meta store, while `ENDPOINT_URL` is the endpoint for the storage service. These two values are distinct and cannot be merged together. Perhaps it would be better to rename `ADDRESS` as `HMS_ADDRESS` instead?

Reply:
if we consider renaming, can we just rename it to HIVE_META_STORE?

Reply:
@Xuanwo can we rename it in a new PR as @soyeric128  before merge this PR?